x-pack/plugins/endpoint/server/plugin.test.ts

```diff
@@ -28,7 +58,10 @@ describe('test endpoint plugin', () => {
       getFeaturesUICapabilities: jest.fn(),
       registerLegacyAPI: jest.fn(),
     };
-    mockedEndpointPluginSetupDependencies = { features: mockedPluginSetupContract };
+    mockedEndpointPluginSetupDependencies = {
+      features: mockedPluginSetupContract,
+      ingestManager: { indexPatternService: new FakeIndexPatternService('') },
```

Comment:
Do you think the following will suffice instead of the actual implementation. 

```
 ingestManager: {
        indexPatternService: ({
          get: jest.fn(() => ''),
        } as unknown) as jest.Mocked<IndexPatternRetriever>,
      }
```

Reply:
Ah yeah, that'll be cleaner, I'll make that change 👍